data/mods/Aftershock/items/afs_bioparts.json

```diff
@@ -90,23 +90,25 @@
     "type": "GUN",
     "name": "vibrating bioblaster",
     "name_plural": "bioblasters",
```

Comment:
Should the plural form also have "vibrating"?

Reply:
You are correct and I needed to change some descriptions to better match other changes.